elyra/pipeline/airflow_template.py

```diff
@@ -0,0 +1,101 @@
+#
+# Copyright 2018-2020 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import json
+import airflow
+from datetime import timedelta
+from airflow import DAG
+from airflow.utils import dates
+from airflow.operators.python_operator import PythonOperator
+from notebook_op import NotebookOp
+
+
+# Function invoked by the python operator
+# Used to construct actual pipeline graph
+def construct_subdag(**kwargs):
+    string_replace = kwargs.get('templates_dict').get("message").replace("'", '"')
+    args = json.loads(string_replace)
+
+    operator_dict = {}
+
+    # Default arguments to use with Operator
+    default_args = {'owner': 'elyra',
+                    'depends_on_past': False,
+                    'start_date': dates.days_ago(1),
```

Comment:
I find this a little confusing, even after googling it (and, to my relief, others do as well).  Would it be possible to either add a comment to this line, or extend the comment on L34 to perhaps indicate what "1 day ago" means?  I suspect it's just how you trigger an immediate schedule dynamically.